### PR TITLE
Fix for issue #3

### PR DESCRIPTION
**Pull request by yoep**
Fixed crash in RageNativeUI when pressing enter on the empty tab item
Code cleanup in the Menus class

Related issue: https://github.com/Albo1125/British-Policing-Script/issues/3

### CODE REVIEW

File: British Policing Script/Menus.cs [hunk 1]
```diff
@@ -1,99 +1,86 @@
...
+            CourtsMenu.AddTab(PublishedResultsList = new TabSubmenuItem("Results", EmptyItems));
 
+            CourtsMenu.RefreshIndex();
         }
+
         private static void OnItemSelect(UIMenu sender, UIMenuItem selectedItem, int index)
         {
-            if (sender == ChecksMenu)
+            if (sender != ChecksMenu)
+                return;
```
Comment (Albo1125) [inline British Policing Script/Menus.cs:69]: Please make use of braces `{` `}` at all times, not doing so makes code more prone to unexpected errors.

File: British Policing Script/Menus.cs [hunk 2]
```diff
@@ -138,41 +127,31 @@ private static void MainLogic()
...
-            }
-        }
+                MenuPool.ProcessMenus();
 
-        private static void ToggleUIMenuEnabled(UIMenu menu, bool Enabled)
-        {
-
-            foreach (UIMenuItem item in menu.MenuItems)
+                if (CourtsMenu.Visible)
+                    CourtsMenu.Update();
```
Comment (Albo1125) [inline British Policing Script/Menus.cs:152]: As above.

**Review by Albo1125 — CHANGES_REQUESTED** [review]
Thanks for the refactoring! Minor changes requested.

File: British Policing Script/Menus.cs (hunk 1 above)
Comment (yoep) [reply British Policing Script/Menus.cs:69]: Updated with braces

File: British Policing Script/Menus.cs (hunk 2 above)
Comment (yoep) [reply British Policing Script/Menus.cs:152]: Updated with braces

**Review by Albo1125 — APPROVED** [review]

**Pull request merged**